src/collection-browser.ts

```diff
@@ -984,7 +1001,7 @@ export class CollectionBrowser
    * If this doesn't change, we don't need to re-fetch the histogram date range
    */
   private get fullQueryNoDateKey() {
-    return `${this.fullQueryWithoutDate}-${this.sortParam?.field}-${this.sortParam?.direction}`;
+    return `${this.fullQueryWithoutDate}-${this.searchType}-${this.sortParam?.field}-${this.sortParam?.direction}`;
```

Comment:
Q: what does this update do?

Reply:
From what I understand, this key is used to determine whether the year histogram needs to be re-fetched. So if someone selects a date facet, the year histogram can remain unchanged because date facets don't affect this key.

However, a different search type *does* affect the need to re-fetch the year histogram (along with all the other results/facets), so it should be part of this key.

Reply:
ah is this the `sin=` support?

Reply:
Not related to the `sin` param support, no. This (and the related change on line 1066) is essentially a UX fix to allow patrons to change the search type and perform a new search without changing the search query. Without this change, collection browser doesn't update the search results if someone just changes the search type (because it relies on these keys to avoid duplicate searches).